Makefile

```diff
@@ -102,6 +102,11 @@ build-benchmarks:
 	@cd benchmark/performanceTest ; GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/PerfTests .
 	@cd benchmark/comparisonTest ;  GO111MODULE=$(GO111MODULE_VALUE) go build -o ../bin/CompTests .
 
+benchmarks-test:
```

Comment:
Can we call this `benchmarks-perf-test` to be clear about which specific tests are running.

Reply:
Updated